test/unit/aggregate_store_test.rb

```diff
@@ -215,6 +313,35 @@ def elasticsearch_store
         assert @instance.aggregate_owner.change_called
       end
 
+      should "keep track of saved changes to the attribute" do
+        passport = sample_passport
+        passport.photo = PassportPhoto.new(color: false)
+        passport.save
+        passport.reload
+        Aggregate::ActiveRecordHelpers::Version.if_version(
+          active_record_gt_4: -> {
+            refute passport.saved_change_to_name?
+            refute passport.photo.saved_change_to_color?
+          },
+          active_record_4: -> {
+            assert_raise(NoMethodError) { passport.saved_change_to_name? }
+            assert_raise(NoMethodError) { passport.photo.saved_change_to_color? }
+          }
+        )
+
+        passport.name = "godzilla"
+        passport.photo.color = true
+        passport.save
+
+        Aggregate::ActiveRecordHelpers::Version.if_version(
+          active_record_gt_4: -> {
+            assert passport.saved_change_to_name?
+            assert passport.saved_change_to_photo?
+            refute passport.photo.saved_change_to_color?
```

Comment:
This mirrors the behavior of `changed?` where changing the color of the photo will show true for `.photo_changed?` but not `.photo.color_changed?`.